desktop/core/src/desktop/lib/raz/raz_client_test.py

```diff
@@ -263,6 +263,36 @@ def test_handle_adls_action_types_mapping(self):
     assert_equal(response['access_type'], 'write')
     assert_equal(response['relative_path'], '/user/csso_hueuser/customers.csv')
 
+    # Chmod
+    method = 'PATCH'
+    relative_path = '/user/csso_hueuser/customers.csv'
+    url_params = {'action': 'setAccessControl'}
+
+    response = client.handle_adls_req_mapping(method, url_params, relative_path)
+
+    assert_equal(response['access_type'], 'set-permission')
+    assert_equal(response['relative_path'], '/user/csso_hueuser/customers.csv')
+
+    # Rename
+    method = 'PUT'
+    relative_path = '/user/csso_hueuser/old_dir' # First call to fetch SAS to sign header path
+    url_params = {}
+
+    response = client.handle_adls_req_mapping(method, url_params, relative_path)
+
+    assert_equal(response['access_type'], 'rename-source')
+    assert_equal(response['relative_path'], '/user/csso_hueuser/old_dir')
+
+    method = 'PUT'
+    relative_path = '/user/csso_hueuser/new_dir' 
+    headers = {'x-ms-rename-source': '/user/csso_hueuser/old_dir?some_sas_token'} # Second call having signed header path
```

Comment:
Not sure it is testing much?
(i.e. the test of the logic of converting the old path to old path with token should leave in the proper unit test file of the class having the logic)